templates/careers/company-culture/index.html

```diff
@@ -125,7 +124,7 @@ <h3 class="p-muted-heading">Committed <br class="u-hide--small">
     </div>
     <div class="col-medium-4 col-9">
       <div class="p-strip is-shallow u-no-padding--top">
-        <p>We aspire to lead on the global stage. Most of us are avid students of greatness &dash; we are interested in how the latest technologies work, how the best organisations work and how the smartest teams collaborate. We aim to hold our own in that crowd. We pursue engineering excellence and love to engineer excellence in all aspects of our work. From releasing the world’s most popular Linux reliably to developing our <a href="/documentation">documentation</a> practice.
+        <p>We aspire to lead on the global stage. Most of us are avid students of greatness &dash; we are interested in how the latest technologies work, how the best organisations work and how the smartest teams collaborate. We aim to hold our own in that crowd. We pursue engineering excellence and love to engineer excellence in all aspects of our work. From releasing the world's most popular Linux reliably to trailblazxing <a href="/documentation">documentation</a>.
```

Comment:
"trailblazing"? Not even sure if the copy doc is right :dancers:

Reply:
I think trailblazing is correct 🙂